src/resolvers/Query/organizations.ts

```
@@ -1,53 +1,29 @@
- * @param args - An object containing `orderBy` and `id` of the Organization.
- * @returns The organization if valid `id` is provided else return organizations with size limit 100.
- * @remarks `id` in the args is optional.
- */
-export const organizations: QueryResolvers["organizations"] = async (
-  _parent,
-  args,
-) => {
-  const sort = getSort(args.orderBy);
 
-  let organizationFound;
+export const organizations = async (_parent, args) => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)